gatorgrouper/tests/test_models.py

```diff
@@ -0,0 +1,95 @@
+"""models.py testing"""
+
+import pytest
+from mixer.backend.django import mixer
+pytestmark = pytest.mark.django_db
+
+
+class TestProfessor:
+    """test professor class"""
+    # pylint: disable=R0201
+    def test_model(self):
+        """test professor model"""
+        obj = mixer.blend('gatorgrouper.Professor')
+        # it creates a professor instance
+        assert obj.pk == 1
+
+    # pylint: disable=R0201
+    def test_str(self):
+        """test __str__ method """
+        obj = mixer.blend('gatorgrouper.Professor', last_name='K', first_name='Greg')
+        result = obj.__str__()
```

Comment:
This should be called with the actual `str` function, like: `result = str(obj)`. That is what the magic method `__str__` enables.

Reply:
Fixed in d3fe167742a644c09feb5d6e682a21c453916028